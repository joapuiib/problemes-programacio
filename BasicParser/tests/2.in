//b//Hola**++ mon//s//
END
