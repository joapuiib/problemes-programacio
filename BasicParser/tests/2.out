\u001B[34mHola\u001B[1m\u001B[0m mon\u001B[39m
